posts/inside-rust/2022-02-11-lang-team-feb-update.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "Lang team February update"
+author: Sean Chen
+description: "Lang team February update"
+team: the lang team <https://lang-team.rust-lang.org/>
+---
+This week the lang team held its February planning meeting ([minutes]). We hold these meetings on the first Wednesday of every month.
```

Comment:
Let's update the date of this blog post (file name) and then also change the wording here, as its not "this week" anymore.

Reply:
I expect we would date the blog post from today.